.ci_support/linux64.yaml

```diff
@@ -17,7 +17,7 @@ cgo_compiler:
 target_platform:
```

Comment:
Please do not modify these files under `.ci_support`. You should only be adding files under the recipe directory that you are building the package for. Additionally any dependencies need to be pulled in from the `conda-forge` channel. If it's not available, then that package needs to be added to conda-forge.

Reply:
@synapticarbors thank you for the feedback.

`gurobi` is a closed-source package from a private company. I think it's very unlikely they will include it in conda-forge since they have their own `gurobi` channel.

Does that mean I won't be able to add my package to conda-forge? If so, do you have any suggestions for uploading it to conda? Maybe I should create my own channel?